.github/workflows/config.yml

```diff
@@ -0,0 +1,66 @@
+name: Test And Deploy Config
+
+on:
+  push:
+    tags:
+      - config
+
+env:
+  CLUSTER: https://api.silver.devops.gov.bc.ca:6443
+  AUTH_TOKEN: ${{ secrets.AUTH_TOKEN_OCP4 }}
+  PROJECT: hcap
+
+jobs:
+  test:
+    name: Test Config
+    runs-on: 'ubuntu-latest'
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Dry run - Dev
+        env:
+          OS_NAMESPACE_SUFFIX: dev
+        run: |
+          oc login --token="$AUTH_TOKEN" --server="$CLUSTER"
+          cd "$GITHUB_WORKSPACE"
+          make server-config-test
+      - name: Dry run - Test
+        env:
+          OS_NAMESPACE_SUFFIX: test
+        run: |
+          oc login --token="$AUTH_TOKEN" --server="$CLUSTER"
+          cd "$GITHUB_WORKSPACE"
+          make server-config-test
+
+  configDev:
+    name: Dev - Update Deployment Config
+    runs-on: ubuntu-latest
+    environment:
+      name: dev
+    env:
+      OS_NAMESPACE_SUFFIX: dev
+    needs: test
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Apply changes
+        run: |
+          oc login --token="$AUTH_TOKEN" --server="$CLUSTER"
+          cd "$GITHUB_WORKSPACE"
+          make server-config
+  configTest:
+    name: Test - Update Deployment Config
+    runs-on: ubuntu-latest
+    environment:
+      name: test
+    env:
+      OS_NAMESPACE_SUFFIX: test
+    needs: test
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Apply changes
+        run: |
+          oc login --token="$AUTH_TOKEN" --server="$CLUSTER"
+          cd "$GITHUB_WORKSPACE"
+          make server-config
```

Comment:
Is reapplying `server-config` going to trigger a deployment config? I thought DC resource would change in K8S and wait for the actual trigger configured in DC or forced via `server-deploy`  my understanding was different. I might be wrong :)

Reply:
Now using same tag to update config. Please review